### ISSUE TITLE
Not supported python3.7

### ISSUE THREAD

**Issue by VictorZuanazzi** [NVIDIAGameWorks/kaolin#10]
Python 3.7 cannot import the pacage. It works all right with python 3.6.9.

Error message:

`Python 3.7.3 (default, Mar 27 2019, 22:11:17) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import kaolin as kal
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/__init__.py", line 17, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/conversions/__init__.py", line 1, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/conversions/meshconversions.py", line 22, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/metrics/__init__.py", line 3, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/metrics/mesh.py", line 40, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/rep/__init__.py", line 1, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/rep/Mesh.py", line 25, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/cuda/load_textures.py", line 7, in <module>
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/site-packages/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg/kaolin/cuda/load_textures.py", line 6, in __bootstrap__
  File "/home/zuanazzi/anaconda3/envs/tomtom/lib/python3.7/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /home/zuanazzi/.cache/Python-Eggs/kaolin-0.2.0+ed13273-py3.7-linux-x86_64.egg-tmp/kaolin/cuda/load_textures.cpython-37m-x86_64-linux-gnu.so: undefined symbol: cudaSetupArgument
`

**Comment by Jean-Francois-Lafleche** [comment]
Hi, I just ran tests on both python 3.7.3 and 3.7.5 and both were successful. Would you be able to please provide the CUDA and pytorch version you are using? Thank you!

**Comment by VictorZuanazzi** [comment]
Python 3.7.3
Pytorch: 1.3.0
Cuda: 10.1-7.6.4

**Comment by Eiii** [comment]
I had a similar issue when trying to install under python3.7.

Install steps:
1. Create & activate virtualenv w/ python3.7
2. `pip install torch`
3. `cd kaolin ; python setup.py install`

Python&CUDA versions:
```
$ python --version
Python 3.7.5
$ pip show torch
Name: torch
Version: 1.3.1
$ nvcc --version
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2019 NVIDIA Corporation
Built on Sun_Jul_28_19:07:16_PDT_2019
Cuda compilation tools, release 10.1, V10.1.243
```

Error:
```
$ python -c "import kaolin"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/__init__.py", line 17, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/conversions/__init__.py", line 1, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/conversions/meshconversions.py", line 22, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/metrics/__init__.py", line 2, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/metrics/point.py", line 17, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/cuda/sided_distance.py", line 7, in <module>
  File "/[venv]/lib/python3.7/site-packages/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg/kaolin/cuda/sided_distance.py", line 6, in __bootstrap__
  File "/[venv]/lib/python3.7/imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: /[cache]/Python-Eggs/kaolin-0.2.0+f858c58-py3.7-linux-x86_64.egg-tmp/kaolin/cuda/sided_distance.cpython-37m-x86_64-linux-gnu.so: undefined symbol: _ZN3c105ErrorC1ENS_14SourceLocationERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE
```

The error goes away when a python3.6 virtualenv is used instead.

**Comment by acivgin1** [comment]
I managed to resolve this error by commenting this line:

```bash
packman/packman pull --platform linux-x86_64 packman/target_deps.xml
```

inside the `buildusd.sh` script. If you read the `target_deps.xml` file you will notice that the `setup.py` procedure downloads the py3.6 files (see [this](https://github.com/NVIDIAGameWorks/kaolin/issues/51#issuecomment-559201958)) and uses py3.6 for building kaolin instead of the python you have active in your environment.

With this, I was able to build kaolin for py3.7 and so far it seems to work without any issues (for my use cases).
